scss/_git_repo.scss

```diff
@@ -1,26 +1,73 @@
+// Variable colors
+$font-color: #676069;
+$border-color: #eee;
+$repo-with: 500px;
+$link-color: #0362d9;
+$button-color: aliceblue;
 .git-repo-areas {
   margin: auto;
 }
 
 .git-repo {
-  .repo-name {
-    font-size: 1.2rem;
+  padding: 25px 0 25px 0;
+  border-bottom: 1px solid $border-color;
+  max-width: $repo-with;
+  .repo-forked {
+    .repo-name {
+      font-size: 1.3rem;
+      text-decoration: none;
+      color: $link-color;
+      font-weight: 700;
+    }
+    span {
```

Comment:
I prefer to use span for text and div for block wrapper but I'm appreciate your comment.